types/express-serve-static-core/ts4.0/express-serve-static-core-tests.ts

```diff
@@ -0,0 +1,161 @@
+import * as express from 'express-serve-static-core';
+
+const app: express.Application = {} as any;
+app.listen(3000);
+app.listen(3000, () => {
+    // no-op error callback
+});
+
+app.get('/:foo', req => {
+    req.params.foo; // $ExpectType string
+    req.params[0]; // $ExpectType string
+    // $ExpectType string | false | null
+    req.is(['application/json', 'application/xml']);
+    // $ExpectType string | false | null
+    req.is('audio/wav');
+    // $ExpectError
+    req.is(1);
+});
+
+app.route('/:foo').get(req => {
+    req.params.foo; // $ExpectType string
+    req.params[0]; // $ExpectType string
+    // $ExpectType string | false | null
+    req.is(['application/json', 'application/xml']);
+    // $ExpectType string | false | null
+    req.is('audio/wav');
+    // $ExpectError
+    req.is(1);
+});
+
+// Params can used as an array
+app.get<express.ParamsArray>('/*', req => {
+    req.params[0]; // $ExpectType string
+    req.params.length; // $ExpectType number
+});
+
+// Params can used as an array - under route
+app.route('/*').get<express.ParamsArray>(req => {
+    req.params[0]; // $ExpectType string
+    req.params.length; // $ExpectType number
+});
+
+// Params can be a custom type
+// NB. out-of-the-box all params are strings, however, other types are allowed to accomadate request validation/coersion middleware
```

Comment:
nit: accommodate (spelling)

Reply:
both this and the misspelling of "coersion" were already there in the original file https://github.com/DefinitelyTyped/DefinitelyTyped/pull/51262/files/59dc475d2ed6032f892a777b8061c11c8d6fbeaa#diff-459c5bd4ded180bff2de08f3ae44f73df64deea0d0ef3989e729876d43daa60c

i fixed it in both files